lib/internal/worker.js

```diff
@@ -496,6 +496,7 @@ function eventLoopUtilization(util1, util2) {
 }
 
 module.exports = {
+  kHandle,
```

Comment:
Is there a specific reason for expose `kHandle` here and mocking`takeHeapSnapshot` it in the test file? It seems like this should work fine with actual data as well

Reply:
I only exported this symbol to mock the takeHeapSnapshot function. Is there other way I can do it?

Reply:
I think you can omit the mocking of `takeHeapSnapshot` instead :slightly_smiling_face: The original method should always work as long as the worker thread is still running.

Reply:
Perfect! Fixed it then!